docs/source/host_challenge.md

```diff
@@ -162,6 +162,51 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
 
 **Congratulations!** you have submitted your challenge configuration for review and [EvalAI team](https://eval.ai/team) has notified about this. [EvalAI team](https://eval.ai/team) will review and will approve the challenge.
 
+## Host a challenge with remote evalution
+
+The steps related to challenge creation will be similar to the steps mentioned above for different types of challenges. This section specifically deals with the bits and pieces required to set up remote challenge evaluation on your own instances.
+
+### Step 1: Modify challenge configuration
+
+Set the `remote_evaluation` parameter to `True` in [`challenge_config.yaml`](https://github.com/Cloud-CV/EvalAI-Starters/blob/621f0cb37b2f1951613c9b6c967ce35be55d34c8/challenge_config.yaml#L12). The `challenge_config.yaml` file defines all the different settings of a challenge and you can edit other things based on your requirement.
+
+Make sure that following fields are set correctly:
+
+- `remote_evaluation` is set to `True`
+
+This will ensure that the challenge worker is aware that the evaluation is to be performed remotely.
+
+### Step 2: Fetch details for the challenge
```

Comment:
This step should be part of setting up remote evaluation script. Please move this point to that section